allennlp/experiments/registry.py

```diff
@@ -209,3 +210,24 @@ def get_token_indexer(cls, name: str) -> Type[TokenIndexer]:
         """
         import allennlp.data.token_indexers  # pylint: disable=unused-variable
         return cls._token_indexers[name]
+
+    @classmethod
+    def list_regularizers(cls) -> List[str]:
+        """
+        Returns a list of all currently-registered :class:`Regularizer` names.
+        """
+        import allennlp.training.regularizers  # pylint: disable=unused-variable
+
+        return _get_keys_with_default(cls._regularizers, "regularizer", cls.default_regularizer)
```

Comment:
Can you add tests for these?  Also, not sure the blank line above is helping anything.